mkdocs_simple_plugin/plugin.py

```diff
@@ -467,27 +465,40 @@ def extract_from(self, from_directory, name, destination_directory):
         for item in self.semiliterate:
             name_match = item['pattern'].search(name)
             if name_match:
-                new_name = (name[:name_match.start(name_match.lastindex)] +
-                            '.md' +
-                            name[name_match.end(name_match.lastindex):])
                 if 'destination' in item:
                     new_name = name_match.expand(item['destination'])
+                else:
+                    if not name_match.lastindex:
+                        raise LookupError(
+                            "mkdocs-simple-plugin: No last group in match of" +
+                            "{} to {} and no destination".format(
+                                item['pattern'], name))
+                    new_name = (name[:name_match.start(name_match.lastindex)] +
```

Comment:
I think it would be clearer if this was something like 

```python
new_name = os.path.splitext(name)[0]+'.md'
```

If someone wants to change the filename to something other than the same file with '.md' extension, they should probably just use the destination field.

Reply:
Seems reasonable to me, and that will eliminate the need for there to be a matching group in the pattern when there's no definition. I will change to your suggested behavior and update the docs.

Reply:
I also added descriptions of the behaviors of the default semiliterate blocks.